cmd/new.go

```diff
@@ -14,11 +14,12 @@ type New struct {
 	*cobra.Command
 
 	// cli args
-	DryRun    bool
-	Component string
-	Kind      string
-	Body      string
-	Custom    []string
+	DryRun     bool
+	Component  string
+	Kind       string
+	Body       string
+	BodyEditor bool // edit body with editor
```

Comment:
```suggestion
	BodyEditor bool
```
This comment is just reiterating the code so we can remove it.